### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: Buttons  and link are not accessible with keyboard in Azure Explorer- Show Properties dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1444]
**Repro Steps:**

1. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
2. Navigate to and select "Show properties" menu item .
3. Properties window will open.
4. Navigate and try to access the  "Overview" ," App settings" Expand collapse buttons ,"Get publish profile" ,"Save"," Discard"," Add", " Remove ", " Edit" buttons  and link  with keyboard.

**Expected Result:**
"Overview"," App settings" Expand collapse buttons ,"Get publish profile" ,"Save"," Discard"," Add"," Remove"," Edit" buttons  and link should be accessible with keyboard in Azure Explorer- Show Properties dialog.

**Actual Result:**
"Overview"," App settings" Expand collapse buttons ,"Get publish profile" ,"Save"," Discard"," Add"," Remove"," Edit" buttons  and link are not accessible with keyboard in Azure Explorer- Show Properties dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![buttons are not accessible with keyboard in properties window](https://user-images.githubusercontent.com/22956130/35729349-33511ce6-07c3-11e8-9049-af9a92dc72d2.png)

**Comment by jdneo** [comment]
Reference:
https://youtrack.jetbrains.com/issue/IDEA-191315

**Comment by Eskibear** [comment]
External issues tracked in #1627

**Comment by jdneo** [comment]
* **Importance of experience**: minimal. very Low usage based on telemetry
* **Quality of experience**: not impacted. Users can view Web App properties in Azure Portal. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by jdneo** [comment]
Workaround:

The same operations can be achieved in Azure Portal:

![image](https://user-images.githubusercontent.com/6193897/43813175-1131ebe6-9af7-11e8-9982-502d1f07563d.png)

![image](https://user-images.githubusercontent.com/6193897/43813191-24b9bdba-9af7-11e8-9436-0c5d9fa785b9.png)

**Comment by WiproPSAccTest** [comment]
Agreed with the resolution , as there is a work around that user can achieve same operation in the Azure portal.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749895
